examples/.cargo/config.toml

```diff
@@ -15,7 +15,7 @@ rustflags = [
 ]
 
 [target.'cfg(target_arch = "xtensa")']
-runner    = "espflash flash --monitor"
+runner    = "espflash flash --monitor --baud 1000000"
```

Comment:
I have set ESPFLASH_BAUD to 2000000 in my environment and would like to get that used (I _think_ espflash will prefer the command line parameter if it's given)